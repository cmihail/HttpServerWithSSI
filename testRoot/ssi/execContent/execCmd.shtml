<html>
<head>
	<title>SSI Exec</title>
</head>

<body>
<!--#exec cmd="ls testRoot/ssi/execContent" -->
</body>
</html>
