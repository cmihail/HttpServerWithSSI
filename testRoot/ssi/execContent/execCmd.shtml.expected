<html>
<head>
	<title>SSI Exec</title>
</head>

<body>
execCmd.shtml
execCmd.shtml.expected

</body>
</html>
